### PR TITLE
Rewrite 'When' page

### LINKED ISSUE #287: "When do I need Vdirsyncer" lists only advantages of git vs. vdirsyncer
The documentation at https://vdirsyncer.readthedocs.org/en/stable/when.html gives no advantage of vdirsyncer compared to git. So, the question "When do I need Vdirsyncer" is not answered.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/27757942-when-do-i-need-vdirsyncer-lists-only-advantages-of-git-vs-vdirsyncer?utm_campaign=plugin&utm_content=tracker%2F574345&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F574345&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by untitaker**
Fix #287

### CODE REVIEW

**Comment by untitaker** [comment]
cc @damiencassou

File: docs/when.rst [hunk 1]
```diff
@@ -30,61 +21,36 @@ Merging the two task lists is left to the user.
 A better idea would've been to use ``git`` to synchronize the ``todo.txt``
 file, which is at least able to resolve some basic conflicts.
```
Comment (hobarrera) [inline docs/when.rst]: Also worth mentioning the lack of clients for lots of devices/platforms, while *dav support is kinda universal.

Comment (untitaker) [inline docs/when.rst]: todo.txt actually has multiple Android apps and at least one iOS app, so I don't think DAV has any advantage there.

On 2 February 2016 00:10:42 CET, Hugo Osvaldo Barrera <notifications@github.com> wrote:
>> @@ -30,61 +21,36 @@ Merging the two task lists is left to the user.
>>  A better idea would've been to use ``git`` to synchronize the
>``todo.txt``
>>  file, which is at least able to resolve some basic conflicts.
>
>Also worth mentioning the lack of clients for lots of
>devices/platforms, while *dav support is kinda universal.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/untitaker/vdirsyncer/pull/327/files#r51497092

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by DamienCassou** [comment]
much better, thanks.

**Pull request merged**